src/models/Fault.js

```diff
@@ -62,6 +68,9 @@ export default class Fault {
             if (data.hasOwnProperty('type')) {
                 obj['type'] = ApiClient.convertToType(data['type'], 'String')
             }
+            if (data.hasOwnProperty('arguments')) {
+                obj['arguments'] = ApiClient.convertToType(data['arguments'], Object)
```

Comment:
If we are going to by consistent in the way our type coercion is done in the rest of the models the correct argument to pass into `convertToType` should be `{String: 'Blob'}` or `{String: Object}`. I would try these to ensure they work, but looking at the code it should work no problem. Doing it this way is a little bit more descriptive on the data's property type.

Reply:
Done 👍